pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=40.2.0", "wheel"]
+requires = ["setuptools>=40.2.0,<46.0.0", "wheel"]
```

Comment:
Why?

Reply:
Or rather, what's using this? https://github.com/pypa/setuptools/issues/2017#issuecomment-605354361

Reply:
Maxmindb

Reply:
Could you provide more detail? And preferably in comment form.

Reply:
Yeah I can

Reply:
Ok, I can't seem to reproduce this now, so just going to remove the constraint and see if it flares up later.